packages/schematics/angular/application/files/src/index.html.template

```diff
@@ -7,6 +7,7 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+
```

Comment:
remove?

Reply:
I assume you mean the extra line space here?